src/com/tigersale/userInterface/customerUser/CustomerUserHomeView.java

```diff
@@ -55,11 +62,22 @@ public void runCustomerUserHomeView()
                 continue;
             }
 
+            Scanner scanner = new Scanner(System.in);
             switch (choice)
             {
                 // Quit the application
                 case 0:
+                    (new WelcomeView(scanner)).runWelcomeView();
```

Comment:
This should not be spawning a new welcome view.  The welcome view is right above this all in the stack.  all you need to do is return